src/blockchain/sync-augur-node-with-blockchain.ts

```diff
@@ -55,8 +55,22 @@ function monitorEthereumNodeHealth(augur: Augur) {
   }, 5000);
 }
 
+const getHttpUrl = (network: NetworkConfiguration) => {
```

Comment:
This isn't necessary.

NetworkConfiguration will read from the environment by default, so if you run without a NetworkConfiguration, `network.ws === process.env.ETHEREUM_HTTP`.

Just use ETHEREUM_HTTP and ETHEREUM_WS in place of the old ENDPOINT_HTTP and ENDPOINT_WS.